.github/workflows/markdown-links.yml

```diff
@@ -13,4 +13,5 @@ jobs:
     - uses: actions/checkout@master
       with:
         fetch-depth: 1
-    - uses: planetscale/github-action-markdown-link-check@master
+        folder-path: docs/
+    - uses: gaurav-nelson/github-action-markdown-link-check@v0.6.0
```

Comment:
swapped back from the fork we were using after the bug got fixed. This is the same one we are using in master anyhow.